core/config/toml/types.go

```diff
@@ -1275,3 +1277,29 @@ func (t *ThresholdKeyShareSecrets) validateMerge(f *ThresholdKeyShareSecrets) (e
 
 	return err
 }
+
+type Tracing struct {
+	Enabled         *bool
+	CollectorTarget *string
+	NodeID          *string
+	Attributes      *map[string]string
```

Comment:
Would `omitempty` help here?
```suggestion
	Attributes      *map[string]string `toml:",omitempty"`
```

Reply:
Oh and we typically group these down below the regular fields rather than mixed in.

Reply:
> Oh and we typically group these down below the regular fields rather than mixed in.

I don't follow - where are you suggesting the `Tracing` struct goes?

Reply:
I mean the `Attributes` field should go at the bottom.